amarachukwu-agbo/hello-books
# #158556732 Enhance server-side tests

#### What does this PR do?
Improve on the existing server-side tests.

#### Description of Task to be completed?
- Add tests for models
- Make test data names more descriptive
- Check for specific messages for 400 error

#### How should this be manually tested?
Clone the repo and install application dependencies. Run command  `npm test` on the console. You should see all the server-side tests passing.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
#158556732

#### Screenshots (if appropriate)
N/A

#### Questions
N/A
